src/ssh/cockpitsshoptions.c

```diff
@@ -100,8 +100,7 @@ cockpit_ssh_options_from_env (gchar **env)
 {
 
   CockpitSshOptions *options = g_new0 (CockpitSshOptions, 1);
-  options->knownhosts_file = get_environment_val (env, "COCKPIT_SSH_KNOWN_HOSTS_FILE",
-                                                  default_knownhosts);
+  options->knownhosts_file = get_environment_val (env, "COCKPIT_SSH_KNOWN_HOSTS_FILE", NULL);
```

Comment:
This could also be a separate commit, and even land now, right? The libssh builtin default for NULL should be exactly what we currently define as `default_knownhosts`. I think this constant and `cockpit_get_default_knownhosts()` should also be dropped, AFAICS we don't use it from anywhere.

Reply:
I'm not sure if it can land right now, first libssh should check the global known_hosts file by default. 

Only then can we stop checking the global known_hosts file explicitly post 0.8 (and stop using `ssh_options->knownhosts_file` for both global known_hosts file and a value set through the env).